packages/apps/shopify-api/runtime/http/headers.ts

```diff
@@ -97,6 +163,6 @@ export function flatHeaders(
   return Object.entries(headers).flatMap(([header, values]) =>
     Array.isArray(values)
       ? values.map((value): [string, string] => [header, value])
-      : [[header, values]],
+      : [[header, values as string]],
```

Comment:
I'm surprised we need this cast, I'd assume this would already be interpreted as a string. Not a problem if it has to stay.